meta.go

```diff
@@ -0,0 +1,162 @@
+package gantry // import "github.com/ad-freiburg/gantry"
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+const (
+	KeepAlive_Yes ServiceKeepAlive = iota
+	KeepAlive_No
+	KeepAlive_Replace
+
+	Log_Stdout ServiceLogHandler = iota
```

Comment:
Above you use CamelCase but the logs are Capitalized_Snake_Case

Reply:
Switched to CamelCase